src/main/java/com/klarna/hiverunner/StandaloneHiveServerContext.java

```diff
@@ -158,10 +156,6 @@ protected void configureAssertionStatus(HiveConf conf) {
                 false);
     }
 
-    protected void configureCheckForDefaultDb(HiveConf conf) {
```

Comment:
Again, just confirming, using HiveRunner with previous versions of Hive will work without this? If so why was this here in the first place?